starwars/src/components/Starwarscharacters.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Character from "./Character";
+
+ function CharacterList(props) {
+  return (
+    <div>
```

Comment:
This would be a better candidate for `ul` and `li`. It's also more semantic.
Ex:
```
<ul>
  {props.characters.map(character => (
     <li><Character character={character} key={character.created} /></li>
   ))}
</ul>
```